classes.py

```diff
@@ -25,6 +25,10 @@
 from .log import *
 from .utils import ProtocolError  # Compatibility with PyLink 1.x
 
+__all__ = ['User', 'IrcUser', 'UserMapping', 'PyLinkNetworkCore',
+        'PyLinkNetworkCoreWithUtils', 'IRCNetwork', 'Irc', 'Server',
+        'IrcServer', 'Channel', 'IrcChannel', 'PUIDGenerator']
+
```

Comment:
Some of these (Irc, IrcServer, IrcChannel, IrcUser) are deprecated aliases and probably don't need to be included here

Actually, they should probably be removed entirely if nothing else depends on them.

Reply:
I noticed there were aliases like this in other places. Maybe removing them should be done in another pull request.

Reply:
Let's add ChannelState to this as well.
And can you fix the ident? Should be like 
```python
__all__ = ['User', '...',
           'PyLinkNetworkCoreWithUtils', '...']
```